src/applications/edu-benefits/complaint-tool/config/form.js

```diff
@@ -387,40 +351,40 @@ const formConfig = {
           uiSchema: {
             educationDetails: {
               school: {
-                facilityCode: {
+                facilityCode: { // Can we unnest this?
```

Comment:
@rianfowler can we unnest facilityCode.facilityCode?

Reply:
It looks like the nesting is necessary for the custom field?

Reply:
It only needs the string- it doesn't have to be submitted this way.

Reply:
Since we have `view:manualSchoolEntry` perhaps this should be `view:searchSchoolEntry`?

Reply:
That works. Will need to update some selectors

Reply:
Thanks for the heads up, will update accordingly.